internal/route/api/v1/repo/path.go

```diff
@@ -11,21 +11,21 @@ import (
 	"gogs.io/gogs/internal/db/errors"
 )
 
-type GitRepoContent struct {
+type repoContents struct {
```

Comment:
It feels like this should be singluar `repoContent`. Later on in the code we can then return  `[]*repoCpntent` in which case `repoContents` feels a little odd to me.